yarn.lock.license

```diff
@@ -0,0 +1,3 @@
+© 2020 Liferay Inc. <https://liferay.com>
```

Comment:
Missing `SPDX-FileCopyrightText` here?

And again, I don't see how we could possibly claim copyright over a file that was:

- Generated
- by a third-party tool 
- derived from a set of trivial configuration files (package.json files).

Reply:
@silverhook might or might not be able to answer this.